yarn.lock

```diff
@@ -899,6 +906,11 @@
     lodash "^4.17.13"
     to-fast-properties "^2.0.0"
 
+"@cennznet/cennznut-wasm@^0.0.1-dev.0":
```

Comment:
wondering why did it change the lock file.. as no dependency is modified..

Reply:
Had other edits in this branch, will remove.